python/fledge/services/core/api/support.py

```diff
@@ -160,6 +166,9 @@ async def get_syslog_entries(request):
         template = __GET_SYSLOG_CMD_TEMPLATE
         lines = __GET_SYSLOG_TOTAL_MATCHED_LINES
         non_total_template = __GET_SYSLOG_TEMPLATE_WITH_NON_TOTALS
+
+        levels = ""
+        level = "debug"
```

Comment:
By default level should be ALL.
And levels = "(INFO|WARNING|ERROR|FATAL|DEBUG)"

Reply:
Do u mean "request.query['level']" would have value ALL in this case? ALL also means "debug and above", correct?

I am using level just as a string to put into /tmp/fledge_syslog_factor file. So it has no more meaning than that.

And I am using levels to compose grep pattern. And setting it to "(INFO|WARNING|ERROR|FATAL|DEBUG)" would just mean unnecessarily complicating grep's work with no benefit.

Reply:
Okay